crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -248,7 +248,7 @@ pub struct StandardMaterial {
     /// | Flint Glass     | 1.69                 |
     /// | Ruby            | 1.71                 |
     /// | Glycerine       | 1.74                 |
-    /// | Saphire         | 1.77                 |
+    /// | Sapphire         | 1.77                 |
```

Comment:
```suggestion
    /// | Sapphire        | 1.77                 |
```

Keep the table aligned.